tracer/test/Datadog.Trace.Security.Unit.Tests/FuzzEncoder.cs

```diff
@@ -9,77 +9,70 @@
 using System.IO;
 using System.Text;
 using Datadog.Trace.AppSec.Waf;
-using Datadog.Trace.AppSec.Waf.Initialization;
 using Datadog.Trace.AppSec.Waf.NativeBindings;
+using Datadog.Trace.Security.Unit.Tests.Utils;
 using Datadog.Trace.TestHelpers;
 using Datadog.Trace.Vendors.Newtonsoft.Json;
 using Datadog.Trace.Vendors.Newtonsoft.Json.Linq;
 using Xunit;
 using Xunit.Abstractions;
 using Encoder = Datadog.Trace.AppSec.Waf.Encoder;
 
-namespace Datadog.Trace.Security.Unit.Tests
+namespace Datadog.Trace.Security.Unit.Tests;
+
+public class FuzzEncoder : WafLibraryRequiredTest
 {
-    [Collection("WafTests")]
```

Comment:
Do these still need to be run not-in-parallel? Or is that no longer an issue?

Reply:
Actually no! Removed all in a13e5682a8421187c2a72718420e9b5f19ba8318